milli/src/update/index_documents/transform.rs

```diff
@@ -205,8 +191,7 @@ impl<'a, 'i> Transform<'a, 'i> {
             // it, transform it into a string and validate it, and then update it in the
             // document. If none is found, and we were told to generate missing document ids, then
             // we create the missing field, and update the new document.
-            let mut uuid_buffer = [0; uuid::fmt::Hyphenated::LENGTH];
-            let external_id = if primary_key_id_nested {
+            if primary_key_id_nested {
```

Comment:
I think we can get rid of this part too and delete the `flatten_from_field_mapping` function!
We'll flatten the document later with the function that works with the field id map :grin: 

Also, in case we need to keep this if, the comment above is outdated

Reply:
Hum... Thank you indeed, it maybe emerged from the rebase I did on main.

Reply:
I have removed this section and the code that supported it :)